ROADMAP.md

```diff
@@ -11,8 +11,14 @@ The project is functional, and moving forward we expect minimal churn in
 the API itself as we complete the implementation of language specific
 features.
 
+This roadmap is influenced by community prioritisation exercises. The
```

Comment:
[question] No mention of USD resolver here, is that because the roadmap is specifically targeted at the core API?

(I feel like I've asked this question before)

Reply:
Doh! No. Good spot, thanks, [removed](https://github.com/OpenAssetIO/OpenAssetIO/pull/1116/commits/5062de944acc608e5d9697fabdff6b57dcd5461d).

Reply:
^ I think this fixup was meant for the other comment I posted.

Reply:
Doh! DOH! Yes.

Re: USD, Yes, that was the thinking.